lib/db/feature-toggle-store.js

```diff
@@ -137,6 +141,29 @@ class FeatureToggleStore {
                 logger.error('Could not archive feature, error was: ', err)
             );
     }
+
+    _importFeature(data) {
+        data = this.eventDataToRow(data);
```

Comment:
It's bad practice to mutate method fields. Please use a new `const` to hold the transformed data object. E.g `const rowData = this.eventDataToRow(data)`

Reply:
understood. will fix it.